plugins/gameobjects/dom/inputtext/InputText.d.ts

```diff
@@ -18,7 +18,7 @@ declare namespace InputText {
         tooltip?: string,
         readOnly?: boolean,
         spellCheck?: boolean,
-        autoComplete?: 'on' | 'off',
+        autoComplete?: string,
```

Comment:
According to the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete), it seems that there can be values other than just “on” or “off” for the autocomplete attribute.

I suggest changing the type to “string”, similar to how it’s defined within the HTMLInputElement interface.

<img width="1133" alt="Screenshot 2023-06-05 at 21 21 40" src="https://github.com/rexrainbow/phaser3-rex-notes/assets/9103819/f4430e14-d73d-4a0a-aff4-ab2538615f7b">

This will allow for a wider range of possible values.